data/hacks.json

```diff
@@ -38,5 +38,13 @@
     "location" : "Santa Clara, CA, USA",
     "website"  : "http://wilhacks.com/",
     "logo"     : "https://challengepost-s3-challengepost.netdna-ssl.com/photos/production/challenge_photos/000/529/752/datas/full_width.png"
+  },
+  {
+    "name" : "Coding Da Vinci",
+    "start" : "21/10/2017",
+    "end" : "2/12/2017",
```

Comment:
@Dumbaz Thanks for the PR can you please make it `02/12/2017`

Reply:
Sure, give me a sec. Also, this is of course in reference to #1, but I assume you´d also take additions to conferences.json?

Reply:
@Dumbaz Yes please have a look at #4 & #5